python_package/brainflow/__init__.py

```diff
@@ -2,4 +2,6 @@
 from brainflow.exit_codes import *
 from brainflow.data_filter import *
 from brainflow.ml_model import *
-from brainflow.utils import *
\ No newline at end of file
+from brainflow.utils import *
+
+__version__ = "0.0.1"
```

Comment:
version is set in CI scripts, they parse setup.py and override its content. For consistency I would recommend to remove it from here or parse it also in CI and override for publishing

Reply:
Removed the version setting from init.py to maintain consistency with CI scripts.